### PR TITLE
fixed minor things

### PR DESCRIPTION
**Pull request by AlekhyaArasavalli**
The following things are addressed:
Id we may not need as we will have default one-->resloved
date columns(From & To) should be implemented without time. Only date we should be accepting-->onlu date is taken
invite code(Any chars accepted) needs to be added in Program--> added
response data needs to returned back for donor POST call-->resolved

### CODE REVIEW

File: Backend/routes/routes.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ async def create_donor(donor: Donor):
     _id = DonorsCollection.insert_one(dict(donor))
     donor = donors_list_serializer(
         DonorsCollection.find({"id": _id.inserted_id}))
-    return {"status": "ok", "data": donor}
+    return donor
```
Comment (mjayakumarmca) [inline Backend/routes/routes.py]: we need to return status + data as a standard I think. Could  you please revert this so that there is consistency.

**Review by mjayakumarmca — CHANGES_REQUESTED** [review]
Also you can add PR description with details of fix for easy reference

File: Backend/routes/routes.py (hunk 1 above)
Comment (AlekhyaArasavalli) [reply Backend/routes/routes.py]: resolved

**Review by mjayakumarmca — APPROVED** [review]
LGTM

**Pull request merged**